src/util/general.js

```diff
@@ -0,0 +1,144 @@
+'use strict'
+
+const _ = require('lodash')
+
+const omitNil = (object) => {
+  return _.omitBy(object, _.isNil)
+}
+
+const pick = (object, properties) => {
+  return _.pick(object, properties)
+}
+
+const assign = (target, source) => {
+  return Object.assign(target, source)
+}
+
+const merge = (target, source) => {
+  return Object.assign({}, target, source)
+}
+
+const mergeAndOmitNil = (target, source) => {
+  return omitNil(merge(target, source))
+}
+
+const formatAmount = (amount, scale = 2) => {
+  return Number(amount).toFixed(scale).toString()
+}
+
+const parseJson = (value) => {
+  if (typeof value !== 'string') {
+    return value
+  }
+
+  try {
+    return JSON.parse(value)
+  } catch (e) {
+    return value
+  }
+}
+
+const squish = (array) => {
+  return _.join(array, '|')
+}
+
+const expand = (value) => {
+  return (value) ? _.split(value, '|') : value
+}
+
+const filterUndefined = (fields) => {
+  for (const key in fields) {
+    if (fields[key] === undefined) {
+      delete fields[key]
+    }
+  }
+  return fields
+}
+
+/**
```

Comment:
Quite a few TODO's here. 

May I suggest that we create a story to do these with the details and remove it from here?

Reply:
These were from the other services, will look at doing them or removing the TODOs @elnyry

Reply:
I agree on creating a story or adding it to the part 2 story

Reply:
Thanks Rajiv.. Having as part of the second part sounds good.

Reply:
Added @elnyry to story https://github.com/mojaloop/project/issues/871